tests/foreman/api/test_contentview.py

```diff
@@ -188,9 +177,7 @@ def test_positive_add_custom_content(self, module_product, module_org, module_ta
 
         :expectedresults: Custom content assigned and present in content view
 
-        :CaseLevel: Integration
-
-        :CaseImportance: Critical
+                :CaseImportance: Critical
```

Comment:
It looks like there are instances where additional indentation was added. I won't comment on each instance in this PR